<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="wjn" name="yuansheng_houtai_action.tuiguang_guanjianciTest" tests="3" failures="0" timestamp="12 Dec 2017 07:30:35 GMT" time="37.480" errors="1">
  <testcase name="dianji" time="19.919" classname="yuansheng_houtai_action.tuiguang_guanjianciTest"/>
  <testcase name="tianjia" time="13.926" classname="yuansheng_houtai_action.tuiguang_guanjianciTest">
    <error type="java.lang.RuntimeException" message="Can&#039;t invoke method public java.lang.Object[][] dataprovider.NSDataProvider1.getData() on this instance of class yuansheng_houtai_action.tuiguang_guanjianciTest due to class mismatch">
      <![CDATA[java.lang.RuntimeException: Can't invoke method public java.lang.Object[][] dataprovider.NSDataProvider1.getData() on this instance of class yuansheng_houtai_action.tuiguang_guanjianciTest due to class mismatch
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:74)
	at org.testng.internal.MethodInvocationHelper.invokeDataProvider(MethodInvocationHelper.java:136)
	at org.testng.internal.Parameters.handleParameters(Parameters.java:430)
	at org.testng.internal.Invoker.handleParameters(Invoker.java:1270)
	at org.testng.internal.Invoker.createParameters(Invoker.java:985)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1075)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at org.testng.TestRunner.privateRun(TestRunner.java:774)
	at org.testng.TestRunner.run(TestRunner.java:624)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:359)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:354)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:312)
	at org.testng.SuiteRunner.run(SuiteRunner.java:261)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1215)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1140)
	at org.testng.TestNG.run(TestNG.java:1048)
	at org.testng.TestNG.privateMain(TestNG.java:1355)
	at org.testng.TestNG.main(TestNG.java:1324)
]]>
    </error>
  </testcase> <!-- tianjia -->
  <testcase name="login1" time="3.635" classname="yuansheng_houtai_action.tuiguang_guanjianciTest"/>
</testsuite> <!-- yuansheng_houtai_action.tuiguang_guanjianciTest -->
